ui/raidboss/data/02-arr/dungeon/brayfloxs_longstop.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.BrayfloxsLongstop,
+  triggers: [
+    {
+      id: 'Brayflox Normal Numbing Breath',
+      netRegex: NetRegexes.startsUsing({ id: '1FA', source: 'Great Yellow Pelican' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Esuna Pelican Poison
+      // This covers cases were multiple people get the poison, including yourself.
+      id: 'Brayflox Normal Pelican Poison Healer',
+      netRegex: NetRegexes.gainsEffect({ effectId: '12' }),
```

Comment:
This 0x12 (in hex, so 18 in decimal) effect id is https://xivapi.com/Status/18?pretty=true.  Can you add `"Poison": "18"` to the `known_mapping` table at the top of `util/gen_effect_id.py` and re-run that script?

Reply:
No need to change gen_effect_id.py then quis?